Remove Flutter v1 Embedding

Error when running project that is depends on ` auto_start_flutter: ^0.1.3`

```
C:\Users\Iqmal\AppData\Local\Pub\Cache\hosted\pub.dev\auto_start_flutter-0.1.3\android\src\main\java\co\techFlow\auto_start_flutter\AutoStartFlutterPlugin.java:9: error: cannot find symbol
import io.flutter.plugin.common.PluginRegistry.Registrar;
                                              ^
  symbol:   class Registrar
  location: interface PluginRegistry
1 error

FAILURE: Build failed with an exception.
```

--- a/android/src/main/java/co/techFlow/auto_start_flutter/AutoStartFlutterPlugin.java
+++ b/android/src/main/java/co/techFlow/auto_start_flutter/AutoStartFlutterPlugin.java
@@ -6,7 +6,6 @@ import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.MethodChannel;
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
 import io.flutter.plugin.common.MethodChannel.Result;
-import io.flutter.plugin.common.PluginRegistry.Registrar;
 import io.flutter.embedding.android.FlutterActivity;
 import io.flutter.embedding.engine.FlutterEngine;
 import io.flutter.plugin.common.BinaryMessenger;

